react/react1/week2/todo-list/src/App.js

```diff
@@ -0,0 +1,57 @@
+import "./App.css";
+import tasksJson from "./tasks.json";
+import React, { useState } from "react";
+import Header from "./Header";
+import TimeOnWebsiteCount from "./TimeOnWebsiteCount";
+import AddTask from "./AddTask";
+import ToDoList from "./ToDoList";
+
+function App() {
+  const [tasks, setTasks] = useState(tasksJson);
+  //const [addTask, setAddTask] = useState("")
+
+  const handleAddTask = () => {
+    const id = tasks.length + 1;
+
+    let newTask = {
+      id: id,
+      task: "Random Text",
+      due_date: "Thu Sep 16 2017",
+      completed: false,
+    };
+    /* Working Code - Saved for future reference.
+  // This works when addTask and setTask is used
+  const addedTaskItem = tasks.push(newTask)
+  setAddTask(addedTaskItem);
+  */
+
+    setTasks((prev) => [...prev, newTask]);
+  };
+
+  const handleCheck = (id) => {
+    const taskItem = tasks.map((task) =>
```

Comment:
``taskItem`` sounds like a single task, while this is an array of tasks. 
The functionality is implemented well, I am just being picky about naming 😸

Reply:
Yes.. I realize it now. Changed to taskItems